python_modules/libraries/dagster-dg/dagster_dg_tests/cli_tests/test_component_type_commands.py

```diff
@@ -79,6 +79,21 @@ def test_component_type_generate_fails_components_lib_package_does_not_exist() -
         assert "Components lib package `bar._lib` is not installed" in str(result.exception)
 
 
+# ########################
+# ##### DOCS
```

Comment:
would be nice to get `markdown_for_component_type` under test too, at least some basic sanity checks